RevenueCatUI/CustomerCenter/Views/CompatibilityContentUnavailableView.swift

```diff
@@ -39,16 +39,22 @@ struct CompatibilityContentUnavailableView: View {
         if #available(iOS 17.0, *) {
             #if swift(>=5.9)
             if let description {
-                ContentUnavailableView(
-                    title,
-                    systemImage: systemImage,
-                    description: description
-                )
+                ContentUnavailableView {
+                    Label {
+                        titleView
+                    } icon: {
+                        iconView
+                    }
+                }
+                description: { description }
             } else {
-                ContentUnavailableView(
-                    title,
-                    systemImage: systemImage
-                )
+                ContentUnavailableView {
+                    Label {
+                        titleView
+                    } icon: {
+                        iconView
+                    }
+                }
```

Comment:
I'm curious 🤔. Would it be possible to remove the `if` and keep only this?
```
ContentUnavailableView {
    Label {
        titleView
    } icon: {
        iconView
    }
}
description: { description }
```
I don't know if `nil` would have the same effect as `EmptyView()` (which is the default value for the `description` parameter when the parameter is not included)